docs/src/main/sphinx/installation/containers.md

```diff
@@ -96,7 +96,7 @@ environmental variable to `dynamic`.
 $ docker run --name trino -d -p 8080:8080 -e CATALOG_MANAGEMENT=dynamic trinodb/trino
 ```
 
-After connecting to Trino, execute (sql-catalog-management)= statements to
+After connecting to Trino, execute {ref}`sql-catalog-management` statements to
```

Comment:
for future updates... the preferred syntax is closer markdown

```
[](sql-catalog-management)
```

or also with some random title

```
[check out the catalog mgt stuff](sql-catalog-management)
```